modules/browser_object_tabbar.py

```diff
@@ -243,8 +243,6 @@ def wait_for_num_tabs(self, num_tabs: int) -> BasePage:
         Waits for the driver.window_handles to be updated accordingly with the number of tabs requested
         """
         try:
-            self.wait.until(
```

Comment:
is this an auto formatter? each time i do the ruff thing it makes it 3 lines, is that wrong im just curious

Reply:
This is definitely an autoformatter, and I think ours are fighting. Let's get on the same page with Ruff version etc.